### PR TITLE
Create hardwareluxx.ru.xml

### PR DESCRIPTION
**Pull request by kronblase**

### CODE REVIEW

File: src/chrome/content/rules/hardwareluxx.ru.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="hardwareluxx.ru">
```
Comment (J0WI) [inline src/chrome/content/rules/hardwareluxx.ru.xml]: This site has mixed content issues, please add `platform="mixedcontent"`

**Comment by kronblase** [comment]
I just realized that adding platform = "mixedcontent" breaks the script and it no longer works. I do not have enough knowledge to fix it.

File: src/chrome/content/rules/hardwareluxx.ru.xml (hunk 1 above)
Comment (kronblase) [inline src/chrome/content/rules/hardwareluxx.ru.xml]: I just realized that adding **platform = "mixedcontent"** breaks the script and it no longer works. I do not have enough knowledge to fix it.

**Comment by J0WI** [comment]
What script?

**Comment by kronblase** [comment]
hardwareluxx.ru.xml

**Comment by J0WI** [comment]
And what does it *break*?

**Comment by kronblase** [comment]
https not work anymore. Not on Tor Browser, not on Firefox.

**Comment by kronblase** [comment]
Opened. He does not work

**Comment by J0WI** [comment]
The ruleset will be shipped with the next release of HTTPS Everywhere.
Rulesets that have the mixedcontent flag will not be enabled by default in normal browsers when `extensions.https_everywhere.enable_mixed_rulesets` is set to `false`. This is to avoid broken pages. However, in Tor Browser it will be enabled.

**Comment by kronblase** [comment]
I'm sorry but I'm a little confused. There will be two ruleset? One for tor browser with platform = "mixedcontent" and one for a normal browser without platform = "mixedcontent"? Simply now, my ruleset in the folder HTTPSEverywhereUserRules with the value platform = "mixedcontent" opens the http page of the site hardwareluxx.ru.

**Comment by J0WI** [comment]
`platform="mixedcontent"` means that the rule is disabled by default, except in Tor Browser which ignores this attribute.

You can enable all `platform="mixedcontent"` rules in normal Firefox by setting `extensions.https_everywhere.enable_mixed_rulesets` to `true`.

**Comment by kronblase** [comment]
But this addon is not only for Tor Browser. In a regular browser, these ruleset will simply not be active by default and the site will open as http.

**Comment by J0WI** [comment]
Yes, that's true. Otherwise users will end up with a broken site, because of the mixed content blocker.
Users that don't care about this can flip the pref.

**Pull request merged**